_events_past/2014-06-12-1st-jlesc-summer-school.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: page_event
+title: 1st JLESC Summer School
+date: 2016-02-15
+updated:
+navbar: Events
+event_location: Sophia Antipolis, France
+event_start: 2014-06-12
+event_end: 2014-06-13
+event_short_url:
+event_url: http://mescal.imag.fr/membres/arnaud.legrand/research/events/puf_jlpc_workshop_14.php
+wayback_url: http://web.archive.org/web/20151228221113/http://mescal.imag.fr/membres/arnaud.legrand/research/events/puf_jlpc_workshop_14.php
+kind: summer_school
+institute: inria
+permalink:
+page_type: event_page
+---
+
+The Joint Laboratory for Petascale Computing focuses on software challenges found in complex 
+high-performance computers. The Joint Laboratory is based at the University of Illinois at 
+Urbana-Champaign and includes researchers from the French national computer science institute Inria, 
+Argonne National Laboratory, Illinois' Center for Extreme-Scale Computation, and the National Center
+for Supercomputing Applications. The Barcelona Supercomputer Center also recently joined this effort. 
+
+The 11th workshop of the INRIA-Illinois-ANL Joint Laboratory will be held at Sophia Antipolis, 
+on the French riviera, from June 9th to June 11th 2014.
+
+The first summer school of the Joint-Laboratory will follow at the same location on June 12th-13th. 
+Both events are supported by the PUF in the context of the NEXTGN project. 
+The theme of this summer school will be on performance metrics, modeling and simulation of large HPC systems. 
+This summer school is offered to students (mainly PhD students and post-docs although Master students may also be interested) 
+of the collaboration and open to students and faculties outside of the collaboration, if room. 
+We are considering a maximum of 30 students. 
+
+### Schedule
+
+Modern computing systems have become increasingly complex and large scale. 
+Understanding their performance has thus become dramatically more difficult than it has ever been.
+The aim of this summer school is to provide attendees a presentation of modern tools and techniques 
+which allow to study the performances of large HPC systems. Four series of lectures (including hands on) 
+on trace/workload analysis/visualization and system simulation/emulation will be given by domain experts. 
+Although classical API (s.a MPI, CHARM++, OmpSs) will be used to illustrate techniques, the fundamental 
+problems remain identical and interesting to any student working with HPC systems and less classical APIs 
+or applications. 
+
+Thursday, June 12th 2014, 8:30AM-12PM:<br/> HPC Application Tracing and Analysis (J. Labarta)
```

Comment:
Don't use HTML `<br />` in Markdown. For intra-paragraph line breaks, Markdown has two spaces at the end of the line.